pyvera/__init__.py

```diff
@@ -284,10 +284,25 @@ def __init__(self, json_obj, vera_controller):
             else:
                 self.name = 'Vera Device ' + str(self.device_id)
 
-    def get_payload_parameter_name(self, name):
+    @staticmethod
```

Comment:
This is clearer https://rapd.wordpress.com/2008/07/02/python-staticmethod-vs-classmethod/